test/controllers/api/docs_controller_test.exs

```diff
@@ -13,10 +13,10 @@ defmodule HexWeb.API.DocsControllerTest do
 
   @tag :integration
   test "release docs" do
-    user           = HexWeb.Repo.get_by!(User, username: "eric")
-    {:ok, phoenix} = Package.create(user, pkg_meta(%{name: "phoenix", description: "Web framework"}))
-    {:ok, _}       = Release.create(phoenix, rel_meta(%{version: "0.0.1", app: "phoenix"}), "")
-    {:ok, _}       = Release.create(phoenix, rel_meta(%{version: "0.0.2", app: "phoenix"}), "")
+    user = HexWeb.Repo.get_by!(User, username: "eric")
```

Comment:
it's probably better to do it in a separate PR, but we could avoid queries by doing:

```elixir
test "release docs", %{user: user} do
```